frontend/components/FilterContent.vue

```diff
@@ -0,0 +1,240 @@
+<template>
+  <div class="wrapper">
+    <div class="drpDwnBtnSect">
+      <div class="headfields">
+        <CourseSelector class="headfield" />
+        <CitySelector class="headfield" />
+        <PaidSelector class="headfield paidSelector" />
+      </div>
+      <hr />
+      <p class="tagTitleText noselect">
+        # Wybierz najważniejszą technologię z tagów
+      </p>
+      <div class="tagsContainer noselect">
+        <div class="tag"><span>React</span></div>
+        <div class="tag"><span>JS</span></div>
+        <div class="tag"><span>Android</span></div>
+        <div class="tag"><span>PHP</span></div>
+        <div class="tag"><span>HTML/CSS</span></div>
+        <div class="tag"><span>Data Analytics</span></div>
+        <div class="tag"><span>C++</span></div>
+        <div class="tag"><span>Python</span></div>
+        <div class="tag"><span>Game Dev</span></div>
+        <div class="tag"><span>Ruby</span></div>
+        <div class="tag"><span>Java</span></div>
+        <div class="tag"><span>.Net</span></div>
+        <div class="tag"><span>Scala</span></div>
+        <div class="tag"><span>Mobile</span></div>
+        <div class="tag"><span>Testing</span></div>
+        <div class="tag"><span>DevOps</span></div>
+        <div class="tag"><span>Security</span></div>
+        <div class="tag"><span>Scala</span></div>
+      </div>
+      <button v-on:click="dropdown" class="approveFilters">
+        Zatwierdź filtry
+      </button>
+    </div>
+    <div v-on:click="dropdown" class="drpDwnBtn noselect">
+      <p class="drpDwmBtnText">Filtrowanie</p>
+    </div>
+  </div>
+</template>
+<script>
+import PaidSelector from "./PaidSelector.vue";
+import CitySelector from "./CityFieldSelector.vue";
```

Comment:
```suggestion
import PaidSelector from "./PaidSelector";
import CitySelector from "./CityFieldSelector";
```

Reply:
Line below its working
`import CourseSelector from "./CourseFieldSelector";`

Reply:
Not sure why yesterday i had error doing the same thing, but for now is ok!

Reply:
Use `@` alias also

Reply:
Also fixed aliases!